src/Microsoft.Extensions.WebEncoders/EncoderServiceCollectionExtensions.cs

```diff
@@ -17,14 +20,23 @@ public static class EncoderServiceCollectionExtensions
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection"/>.</param>
         /// <returns>The <see cref="IServiceCollection"/> instance after the encoders have been added.</returns>
-        public static IServiceCollection AddWebEncoders(this IServiceCollection services)
+        public static void AddWebEncoders(this IServiceCollection services)
```

Comment:
I understand that's proposed pattern with this change, I'm asking why that is better than preserving chaining and having them just return the serviceCollection?

```
    services.AddRouting()
       .AddLocalization().
       .AddCaching().
       .AddMvc().Services.
       .AddCors()
```

IServiceCollection's other Add methods are all chainable, so why should the framework Add methods be different?